client/public/index.html

```diff
@@ -4,12 +4,17 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
+    <meta name="theme-color" content="#ffffff">
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Pie for Providers uses technology to help child care providers earn more money and to help low-income families access early care and education. We are working for a more equitable early childhood field through fair pay for caregivers, effective government funding systems, and sustainable child care business models."
```

Comment:
This is just a placeholder. @csprayregen Can you update it?

Reply:
What is this for @arku ?

Reply:
@csprayregen [Meta description](https://moz.com/learn/seo/meta-description).